openpower/package/rainier-2u-xml/rainier-2u-xml.mk

```diff
@@ -137,6 +137,19 @@ define RAINIER_2U_XML_BUILD_CMDS
 	    cd $(RAINIER_2U_XML_MRW_SCRATCH) && dd if=/dev/zero of=$(WOF_OVERRIDE_BIN) bs=4096 count=1; \
 	fi
 
+	$(eval PSPD_DATA = $$(patsubst %.xml,%.pspddata,$(call qstrip,$(BR2_RAINIER_2U_XML_FILENAME))))
+	$(eval PSPD_BIN = $$(patsubst %.xml,%.PSPD.bin,$(call qstrip,$(BR2_RAINIER_2U_XML_FILENAME))))
+
+        if [ -e "$(RAINIER_2U_XML_MRW_SCRATCH)/$(PSPD_BIN)" ]; then \
+            rm $(RAINIER_2U_XML_MRW_SCRATCH)/$(PSPD_BIN); \
+        fi
+
+        # if PSPD_DATA exists the PSPD.bin will be built with the data
+        if [ -e "$(RAINIER_2U_XML_MRW_HB_TOOLS)/buildSPDImages.pl" ] && [ -e "$(RAINIER_2U_XML_MRW_SCRATCH)/$(PSPD_DATA)" ]; then \
+            chmod +x $(RAINIER_2U_XML_MRW_HB_TOOLS)/buildSPDImages.pl; \
+            $(RAINIER_2U_XML_MRW_HB_TOOLS)/buildSPDImages.pl --spdOutBin $(RAINIER_2U_XML_MRW_SCRATCH)/$(PSPD_BIN) --ecImg_10 $(RAINIER_2U_XML_MRW_SCRATCH)/$(PSPD_DATA); \
```

Comment:
I don't really like the idea that we have to update this file any time one of the xml repos adds a new file.  Let's try to come up with something better.

Reply:
Pushed up a dynamic flavor for review.